beacon_node/network/src/subnet_service/tests/mod.rs

```diff
@@ -119,7 +118,7 @@ fn get_logger(log_level: Option<slog::Level>) -> Logger {
 static CHAIN: LazyLock<TestBeaconChain> = LazyLock::new(TestBeaconChain::new_with_system_clock);
 
 fn get_subnet_service() -> SubnetService<TestBeaconChainType> {
-    let log = test_logger();
+    let log = get_logger(TEST_LOG_LEVEL);
```

Comment:
Why not use the test_logger? We want to toggle logs programmatically with `--features test_logger`

Reply:
I set it to be consistent with `TestBeaconChain` since it uses `get_logger(TEST_LOG_LEVEL)`. 
Didn't know the `test_logger` feature was a thing. I'll change both to `test_logger()`